tools/datafix/delete_bugs_with_source.py

```diff
@@ -42,6 +42,7 @@ def main() -> None:
 
   query = client.query(kind="Bug")
   query.add_filter(filter=PropertyFilter("source", "=", args.source))
+  query.keys_only()
```

Comment:
Do you intend to print out the full bug when verbose is true? Or just the key, if you want to print out the full bug you probably want to only call this when verbose if false.

Reply:
Good point. Adjusted accordingly in 6631cd5